src/avatar.story.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import Avatar from './Avatar.jsx';
 import { storiesOf, action } from '@kadira/storybook';
-import { MOCK_MEMBER } from 'meetup-web-platform/util/mocks/api';
+import { MOCK_MEMBER } from 'meetup-web-platform/lib/util/mocks/api';
```

Comment:
@mmcgahan is it no longer possible to import the es6 modules? I was getting an error importing `meetup-web-platform/util/mocks/api`. Or is it just that now it would have to be `meetup-web-platform/src/util/mocks/api`?